CHANGELOG.next.asciidoc

```diff
@@ -107,6 +107,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Change logging in logs input to structure logging. Some log message formats have changed. {pull}25299[25299]
 - Change source field for `event.action` in `fortinet.firewall` module to `fortinet.firewall.action` instead of `fortinet.firewall.eventtype`. {pull}24816[24816]
 - threatintel module: Changed the type of `threatintel.indicator.first_seen` from `keyword` to `date`. {pull}26765[26765]
+- Removed all alias fields from the Suricata modules. {issue}10535[10535] {pull}26627[26627]
```

Comment:
looks like you have 2 entries in the Changelog.

Reply:
Thanks. Fixed.